Database/Base/WorldBase.sql

```diff
@@ -21,201 +21,244 @@ USE `ace_world`;
 DROP TABLE IF EXISTS `ace_object`;
 
 CREATE TABLE `ace_object` (
-  `aceObjectId` INT(10) UNSIGNED NOT NULL,
-  `aceObjectDescriptionFlags` INT(10) UNSIGNED NOT NULL,
-  `weenieClassId` INT(10) UNSIGNED NOT NULL,
-  `weenieHeaderFlags` INT(10) UNSIGNED NOT NULL,
-  `physicsDescriptionFlag` INT(10) UNSIGNED NOT NULL,
-  `currentMotionState` TEXT DEFAULT NULL,
+  `aceObjectId` int(10) unsigned NOT NULL,
+  `aceObjectDescriptionFlags` int(10) unsigned NOT NULL,
+  `weenieClassId` int(10) unsigned NOT NULL,
+  `weenieHeaderFlags` int(10) unsigned NOT NULL,
+  `physicsDescriptionFlag` int(10) unsigned NOT NULL,
+  `currentMotionState` text DEFAULT NULL,
   PRIMARY KEY (`aceObjectId`),
-  KEY `idx_weenie` (`weenieClassId`)
-) ENGINE=INNODB DEFAULT CHARSET=utf8;
+  KEY `idx_weenie` (`weenieClassId`),
+  CONSTRAINT `fk_weenie_ao` FOREIGN KEY (`weenieClassId`) REFERENCES `ace_weenie_class` (`weenieClassId`)
```

Comment:
can't have a foreign key before you have the table.  script is not safe to run once and be done.